packages/form-layout/src/vaadin-form-layout-styles.js

```diff
@@ -207,6 +211,11 @@ export const formItemStyles = css`
     flex: 1 1 auto;
   }
 
+  #content[auto-responsive] ::slotted(*) {
```

Comment:
I thought about applying this by default, but I'm unsure if `box-sizing: border-box` is safe to use in non-auto-responsive mode.